src/TimeoutExtender.js

```diff
@@ -52,7 +59,7 @@ class TimeoutExtender {
     this._squiss.on('handled', msg => this.deleteMessage(msg))
     this._squiss.on('message', msg => this.addMessage(msg))
     this._visTimeout = this._opts.visibilityTimeoutSecs * 1000
-    this._stopAfter = this._opts.noExtensionsAfterSecs * 1000
+    this._stopAfter = Math.min(this._opts.noExtensionsAfterSecs * 1000, MAX_MESSAGE_AGE_MS)
```

Comment:
I guess either way, the `Math.min` will still end up with `MAX_MESSAGE_AGE_MS` so maybe you intended to allow this?!